### ISSUE TITLE
Central Marketplace

### ISSUE THREAD

**Issue by eordano** [decentraland/districts#3]
Central Marketplace: original proposal by @rdixon22

Link:

https://github.com/decentraland/proposals/issues/86

**Comment by 404wizard** [comment]
Great idea but you should file a formal proposal as outlined on their slack. I would also change the name to something more interesting like "Grand Bazaar".

**Comment by rdixon22** [comment]
@mkirlew Yes, that formal proposal format was only posted a few hours ago, but I'm on it.

**Comment by rdixon22** [comment]
The Central Marketplace (AKA Grand Bazaar) will let Asset Store vendors present and sell their goods in an interactive 3D space. In fact this project could end up contributing to the overall Asset Store framework within Decentraland.

Please see the original proposal (copied below from [decentraland/proposals#86](https://github.com/decentraland/proposals/issues/86) ) for features and illustrations.

**Title:** **Central Marketplace and Asset Store**

**Description:** To set up a central open-air marketplace where vendors can actively sell in-app goods.

**Project Lead:** Rob Dixon @rdixon22 (rdixon on slack)

**Team:** To be determined

**Goals:**  
1. Provide a public service for new users and anyone who wants to get pre-built assets for their spaces. 
2. Help asset creators get more customers for their wares and build successful secondary businesses in Decentraland. 
3. Cover costs, and maybe provide a future revenue stream, for the developers.
4. Develop a new standard for immersive VR shopping experiences.

**Category**: Publicly accessible. Private (district) ownership.

**LAND Needed:** Small version: 9 LAND (3 by 3, if square); Larger version: 25 LAND (5 by 5). If hex layout is used, small version needs 10 LAND (3-4-3) and larger version needs 19 LAND (3-4-5-4-3).

**Startup Costs:** A full budget will be done soon. Initial assumption is 2 to 3 developers full-time for 3 months to build smart contracts and an initial UI/interaction prototype. The full project is larger, though. Final scope will depend on whether our team takes on the overall Asset Store framework for DCL, or just builds a VR marketplace that uses existing Asset Store contracts from the DCL team.

**How will funds be raised:** Sweat equity, unless a donor is willing to fund this.

**How will contributors see a return value:** If the project is built with sweat equity by the initial team, then the team will share future Asset Store revenues to pay themselves back. Even using very thin transaction fees (for example, 3% of transaction value instead of the 30% used by Apple and Google app stores) future revenues might be strong.

**Attachments / Designs / Budgets:** A full budget will be uploaded once we know the size of the team, and whether we should take on the full Asset Store feature set, or just the 3d marketplace presentation.

---------------
Here is the full proposal from before, for reference:

**Purpose**: To set up a central open-air marketplace where vendors can actively sell in-app goods

**Description**:
It is assumed that there will be some kind of general asset store feature in Decentraland for land owners to buy pre-built 3D models or behaviors (scripts) from other users in order to enhance their lands. (Depending on the eventual player/user model, it's possible that non-land-owners will be allowed to purchase goods as well, such as avatars, costumes, equipment and so on).

One of the best examples to emulate for the overall content market will be the Asset Store for Unity3D -- but that is a flat, web-based, 2D user interface.

This proposal suggests setting up a 3D open-air marketplace in Decentraland to let players browse and purchase products, and interact with vendors. This would be a much more fun experience than just scrolling through 2D lists of products. Plus it would give vendors a way to actively promote their products, and help players discover products they might not know existed.

I'd like to propose a large Market Square area be built near the center of the seed area. A 30m x 30m area would probably be large enough to contain about 18 separate booths, depending on their size and placement.

![palmyra-market-3d-model-low-poly-obj-3ds-fbx-stl-blend-mtl png](https://user-images.githubusercontent.com/1307781/29355361-9f14003e-8225-11e7-8787-e2f6fb6f6e0a.jpeg)
_(One possible style of marketplace that could be used.)_

Vendors can rent booths or tables in this area, much like they would at traditional street markets or farmers markets. Each vendor would select one or more of their products to display at their booth or table. Their entire product list would then be accessible by players through a simple interface (for example, gazing at a designated sign at the booth to open the full product list).

The market could be open 24 hours a day for a given period, say a week or a month at a time. After that the booths would be available to rent by someone else for the next period.

Vendors could be present at their booths now and then, but at other times players could just purchase assets directly from the booths.

Of course, this could also be done as a private enterprise by a land owner, as opposed to it being a public service set up by Decentraland.

Rob (rdixon)

![ancient-market-3d-model-max-fbx](https://user-images.githubusercontent.com/1307781/29355383-b6daad30-8225-11e7-8b0d-b7bcc505fcf1.jpg)
_(I hope our vendors will sell more interesting products than these.)_

**Comment by iefken** [comment]
Hi!
So I've contributed land to you, what now? I can see them in my contributions list but not sure what I can do now...
Any updates?
Thanks!
